src/main/kotlin/io/github/orangain/prettyjsonlog/logentry/Timestamp.kt

```diff
@@ -49,7 +49,7 @@ sealed interface Timestamp {
     }
 }
 
-private val timestampKeys = listOf("timestamp", "time", "@timestamp", "ts", "@t", "Timestamp")
```

Comment:
I guess this change should be reverted in order to parse the pair of `timestampSeconds` and `timestampNanos` correctly.

In addition, could you please add one more test case to [ExtractTest.kt](https://github.com/orangain/pretty-json-log-plugin/blob/v0.4.1/src/test/kotlin/io/github/orangain/prettyjsonlog/logentry/ExtractTest.kt#L16) so that we can confirm that the plugin can extract these fields from an actual JSON log?

Reply:
Have added to ExtractTest.kt and reverted what I could.